specification/resources/resource-manager/readme.md

```diff
@@ -56,6 +56,18 @@ tag: package-links-2016-09
 tag: package-managedapplications-2018-06
 ```
 
+``` yaml $(package-deploymentscripts)
+tag: package-deploymentscripts-2019-10-preview
+```
+
+### Tag: package-resources-2019-10-preview
```

Comment:
Shouldn't this be `package-deploymentscripts-2019-10-preview`?

Reply:
Tag name change is fine with us. It sounds like it'll be more clear this way.